components/omega/src/infra/Broadcast.cpp

```diff
@@ -0,0 +1,184 @@
+//===-- infra/Broacast.cpp - Implements Broadcasting functions --*- C++ -*-===//
+//
+/// \file
+/// \brief implements blocking and non-blocking broadcasting functions
+///
+/// This implements blocking and non-blocking broadcasting functions. Two
+/// function names are overloaded accross Omega data types: I4, I8, R4, R8,
+/// Real, bool, and std::string: 1) Broadcast for blocking mode and 2) mpiIbcast
+/// for non- blocking mode.
+//
+//===----------------------------------------------------------------------===//
+
+#include "Broadcast.h"
+
+namespace OMEGA {
+
+//------------------------------------------------------------------------------
+// Broadcast I4 scalar value
+int Broadcast(I4 &value, const MachEnv *InEnv, const int rankBcast) {
```

Comment:
Following the style guide, all vars start with a capital letter (I still have trouble with this too as evidenced by the design doc). So Value, RankBcast and Retval/RetVal throughout.

Reply:
renamed variables as suggested.